pkgs/development/tools/build-managers/colormake/default.nix

```diff
@@ -17,9 +18,9 @@ stdenv.mkDerivation {
     cp -fa colormake.pl colormake colormake-short clmake clmake-short $out/bin
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Simple wrapper around make to colorize the output";
```

Comment:
Maybe put a homepage in at the same time? also seems to be missing a maintainer. otherwise looks good